lib/runCall.js

```diff
@@ -10,6 +10,7 @@ const ERROR = exceptions.ERROR
  * runs a CALL operation
  * @method runCall
  * @param opts
+ * @param cb the callback
```

Comment:
This should be at the end of the ``opts.xxx`` param values (or again, if this is generally wrong, changed everywhere for consistency).

Reply:
Sure thing